eng/liveBuilds.targets

```diff
@@ -228,12 +228,12 @@
       
       <LibrariesRuntimeFiles Condition="'$(TargetOS)' == 'browser' and '$(RuntimeFlavor)' == 'CoreCLR'"
                              Include="
-        $(CoreCLRNativeArtifactsPath)dotnet.js;
-        $(CoreCLRNativeArtifactsPath)dotnet.js.map;
-        $(CoreCLRNativeArtifactsPath)dotnet.runtime.js;
-        $(CoreCLRNativeArtifactsPath)dotnet.runtime.js.map;
-        $(CoreCLRNativeArtifactsPath)dotnet.d.ts;
-        $(CoreCLRNativeArtifactsPath)package.json;"
+        $(LibrariesNativeArtifactsPath)dotnet.js;
+        $(LibrariesNativeArtifactsPath)dotnet.js.map;
+        $(LibrariesNativeArtifactsPath)dotnet.runtime.js;
+        $(LibrariesNativeArtifactsPath)dotnet.runtime.js.map;
+        $(LibrariesNativeArtifactsPath)dotnet.d.ts;
+        $(LibrariesNativeArtifactsPath)package.json;"
         IsNative="true" />
       <LibrariesRuntimeFiles Condition="'$(TargetOS)' == 'browser' and '$(RuntimeFlavor)' == 'CoreCLR'"
                              Include="$(CoreCLRNativeArtifactsPath)src\es6\*.js"
```

Comment:
For a reason I don't remember anymore I have crafted it this way https://github.com/dotnet/runtimelab/pull/2453/files#diff-e39b13480e09094a7907aaf2c25d8bec6025771736925298c71182d6332f0956. If the correct approach is to use `LibrariesNativeArtifactsPath`, then we can drop definition of `CoreCLRNativeArtifactsPath`

Reply:
Thank you for pointing to that commit - it actually looks wrong to me to `LibrariesNativeArtifactsPath` here, since it picks up the configuration (`Debug`/`Release`) from libraries, while these are runtime files and so should pick up CoreCLR's configuration (as the before-merge code does).

Reply:
Do you want me to revert this and confirm the error ?

Reply:
It would be useful to understand what the bug here is, yes.